### DISCUSSION TITLE
[公告] 游戏运行出现问题先看这里

### DISCUSSION THREAD

**Discussion by wangling12** [Gamuxorg/bbs#220] [公告通知]
若游戏运行出现问题请先尝试用以下方式解决:

1. 检查是否赋予可执行权限
    `chmod a+x 文件名`

2. 检查fuse是否安装
    以apt为例`sudo apt install fuse`

3. 安装`steam`,并至少打开一次
    不论以什么方式,安装steam并打开一次,主要用于确认游戏必须的图形库已经正常安装

**Comment by shouhuanxiaoji** [comment]
后面会有详细文档，说明游戏运行的注意事项。

**Comment by luwenhegithub** [comment]
我用的是manjaro（基于archlinux的发行版），也安装了fuse，但在运行文明5的时候仍然提示找不到libfuse.so.2，后来我发现manjaro的只有64位的libfuse.so.2没有32位的，而文明5需要的是32位的

这个问题应该如何解决呢？

🙏🙏🙏

**Comment by hyh19962008** [comment]
如果arch源没有的话，可以去rpmfind或者ubuntu launchpad直接搜索下载rpm包或者deb包，提取出so文件放到/usr/lib下的对应目录（应该是linux-gnu-i386，不确定）

Best

________________________________

**Comment by shouhuanxiaoji** [comment]
@luwenhegithub 回头会专门解决这个问题，同时如果有条件，以后会在manjaro测试游戏

**Comment by gynamics** [comment]
> 我用的是manjaro（基于archlinux的发行版），也安装了fuse，但在运行文明5的时候仍然提示找不到libfuse.so.2，后来我发现manjaro的只有64位的libfuse.so.2没有32位的，而文明5需要的是32位的
> 
> 这个问题应该如何解决呢？
> 
> praypraypray

Arch可使用pacman从multilib下载32位运行库，参考
https://wiki.archlinux.org/index.php/Official_repositories_(%E7%AE%80%E4%BD%93%E4%B8%AD%E6%96%87)#multilib

**Comment by shouhuanxiaoji** [comment]
文明5现在应该不再缺32位依赖

**Comment by luwenhegithub** [comment]
> 文明5现在应该不再缺32位依赖

👍👍👍

**Comment by nukemiko** [comment]
> 我用的是manjaro（基于archlinux的发行版），也安装了fuse，但在运行文明5的时候仍然提示找不到libfuse.so.2，后来我发现manjaro的只有64位的libfuse.so.2没有32位的，而文明5需要的是32位的
> 
> 这个问题应该如何解决呢？
> 
> 🙏🙏🙏

manjaro应该可以使用multilib源吧，32位的库都在这个源里面

**Reply by nukemiko** [reply]
补充：对于32位的 fuse2/3，你需要通过 AUR 构建安装 [lib32-fuse2](https://aur.archlinux.org/packages/lib32-fuse2)/[lib32-fuse3](https://aur.archlinux.org/packages/lib32-fuse3)。
